docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -119,17 +126,16 @@ Running a CodeQL pack
 
 .. include:: ../reusables/beta-note-package-management.rst
 
-To run an existing CodeQL query pack from the GitHub Container registry, you need to download it first::
-
-   codeql pack download microsoft/coding-standards@1.0.0
+To run an existing CodeQL query pack from the GitHub Container registry, you can specify one or more
+pack names and use the ``--download`` flag::
 
-Afterwards, you can run the pack on a specific database::
+   codeql database analyze --download <database> microsoft/coding-standards@1.0.0 github/secutiry-queries --format=sarifv2.1.0 --output=query-results.sarif
```

Comment:
I'd tend to put the download flag later in the command to reduce the tendency for people to see it as connected in anyway to `<database>` but maybe this is something that no developer would think 🤔